dali/pipeline/data/layouts.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_DATA_LAYOUTS_H_
+#define DALI_PIPELINE_DATA_LAYOUTS_H_
+
+#include "dali/core/tensor_layout.h"
+
+namespace dali {
+
+// Layouts containing spatial dimensions such as height, width, depth
+// Contains Image, Volumetric Images, and Sequence layouts
+static std::initializer_list<TensorLayout> kSpatialLayouts {
+    "HWC", "CHW", "DHWC", "FHWC", "FCHW", "CDHW", "CFHW", "FDHWC", "FCDHW", "CFDHW"
```

Comment:
These are not spatial layouts, since they also include _temporal_ dimension.
Also, I don't like mixing such lists into a high-impact (and therefore, high build-time) header.

Reply:
Removed it